### PR TITLE
Blog post for 2023 Outreachy interns

### PR DESCRIPTION
**Pull request by wetneb**
Here is a blog post to announce the internships. Let me know if you want anything changed (or feel free to edit yourself, via the "Files changed" tab).

cc @lozanaross @Lydiaofficial @ayushrai206

### CODE REVIEW

**Review by Ainali — APPROVED** [review]

File: blog/2023-05-05-outreachy-interns.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+author: Antonin Delpeuch
+title: "Two Outreachy interns join us for the summer"
+---
+
+OpenRefine participates to the [Outreachy](https://outreachy.org) programme, which provides internships to people subject to systemic bias in the technical industry.
```
Comment (tfmorris) [inline blog/2023-05-05-outreachy-interns.md:6]:
```suggestion
The OpenRefine project participates in the [Outreachy](https://outreachy.org) program, which provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.
```

File: blog/2023-05-05-outreachy-interns.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+We are happy to announce that this summer, we will be hosting two more interns, after participating to previous rounds in 2020 and 2022.
```
Comment (tfmorris) [inline blog/2023-05-05-outreachy-interns.md:7]:
```suggestion
We are happy to announce that for the May 2023 cohort, we will be hosting two more interns, after participating to previous rounds in 2020 and 2022.
```

File: blog/2023-05-05-outreachy-interns.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+[Lydia Amadi Chinyere](https://github.com/Lydiaofficial) will be joining us to work on improving design workflows in the project and documenting the design patterns currently followed in the application. She will be mentored by [Lozana Rossenova](https://github.com/lozanaross).
```
Comment (tfmorris) [inline blog/2023-05-05-outreachy-interns.md:9]:
```suggestion
[Lydia Amadi Chinyere](https://github.com/Lydiaofficial) will be joining us to work on improving design workflows in the project and documenting the design patterns currently followed in the OpenRefine application. She will be mentored by [Lozana Rossenova](https://github.com/lozanaross).
```

File: blog/2023-05-05-outreachy-interns.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+[Ayushi Rai](https://github.com/ayushrai206) will help us out with various improvements to our reconciliation features. She will be mentored by [Antonin Delpeuch](https://github.com/wetneb).
+
+We thank all the applicants for their work during the contribution period and encourage them to continue working on free and open source software. The number of interns we can accept is limited by our mentoring capacity. We are also grateful to the [Chan
+Zuckerberg Initiative](https://chanzuckerberg.com/) for funding OpenRefine's participation in Outreachy via a Diversity and Inclusion grant, part of the [Essential Open Source Software for Science](https://chanzuckerberg.com/eoss/) programme.
```
Comment (tfmorris) [inline blog/2023-05-05-outreachy-interns.md:14]:
```suggestion
Zuckerberg Initiative](https://chanzuckerberg.com/) for funding then OpenRefine team's participation in Outreachy via a Diversity and Inclusion grant, part of the [Essential Open Source Software for Science](https://chanzuckerberg.com/eoss/) program. 
```

**Review by tfmorris — CHANGES_REQUESTED** [review]
Minor copy edits for grammar, international English spelling, and brand.

**Review by tfmorris — APPROVED** [review]

**Pull request merged**